Tests/svgLib/path/parser_test.py

```diff
@@ -232,23 +232,28 @@
             [
                 ('moveTo', ((100.0, 100.0),)),
                 ('qCurveTo', ((100.0, 100.0),
-                    (217.1758387643403, 139.78681395054122),
-                    (324.3782932812971, 77.97418174642395),
-                    (348.6469560082834, -43.36913188850798),
-                    (273.464933455753, -141.65865950652255),
+                    (217.17583, 139.78681),
+                    (324.37829, 77.97418),
+                    (348.64695, -43.36913),
+                    (273.46493, -141.65865),
                     (150.0, -150.0))),
                 ('lineTo', ((100.0, 100.0),)),
                 ('closePath', ()),
             ]
         ),
     ]
 )
+
 def test_parse_path(pathdef, expected):
     pen = RecordingPen()
     parse_path(pathdef, pen)
 
-    assert pen.value == expected
-
+    assert len(pen.value) == len(expected)
+    for i, (instr, coords) in enumerate(pen.value):
```

Comment:
i think it's better if you use `zip` to iterate over the two lists instead of enumerate and indexing.
Also you need to assert the tuples of coordinates have the same length.
And it's not "real" and "imag" in the tuples, but x and y floats.
 
```diff
diff --git a/Tests/svgLib/path/parser_test.py b/Tests/svgLib/path/parser_test.py
index f8aa0951..3e001bb7 100644
--- a/Tests/svgLib/path/parser_test.py
+++ b/Tests/svgLib/path/parser_test.py
@@ -249,11 +249,12 @@ def test_parse_path(pathdef, expected):
     parse_path(pathdef, pen)
 
     assert len(pen.value) == len(expected)
-    for i, (instr, coords) in enumerate(pen.value):
-        # instr are 'moveTo', 'qCurveTo', 'closePath' etc.
-        assert instr == expected[i][0]
-        for j, (real, imag) in enumerate(coords):
-            assert pytest.approx((real, imag)) == expected[i][1][j]
+    for (instr, coords), (exp_instr, exp_coords) in zip(pen.value, expected):
+        assert instr == exp_instr
+        assert len(coords) == len(exp_coords)
+        for c, e in zip(coords, exp_coords):
+            assert c == pytest.approx(e)
+
 
 @pytest.mark.parametrize(
     "pathdef1, pathdef2",
```